volume/volume_windows.go

```diff
@@ -94,91 +102,39 @@ func (m *MountPoint) BackwardsCompatible() bool {
 	return false
 }
 
-// ParseMountSpec validates the configuration of mount information is valid.
-func ParseMountSpec(spec string, volumeDriver string) (*MountPoint, error) {
-	var specExp = regexp.MustCompile(`^` + RXSource + RXDestination + RXMode + `$`)
-
+func splitRawSpec(raw string) ([]string, error) {
+	specExp := regexp.MustCompile(`^` + RXSource + RXDestination + RXMode + `$`)
 	// Ensure in platform semantics for matching. The CLI will send in Unix semantics.
-	match := specExp.FindStringSubmatch(filepath.FromSlash(strings.ToLower(spec)))
+	match := specExp.FindStringSubmatch(strings.ToLower(raw))
```

Comment:
Why removing `FromSlash`? If legitimate (which I'm not sure why), the comment immediately above is now irrelevant.